src/src/main/java/ca/bc/gov/open/pssg/docmerge/config/ConfigProperties.java

```diff
@@ -0,0 +1,61 @@
+package ca.bc.gov.open.pssg.docmerge.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * 
+ * Externalize configuration for easy access to properties either as OpenShift Secrets or local Env variables. 
+ * 
+ * @author shaunmillargov
```

Comment:
if this was modified it should be , alexjoy, taygov ....etc

Reply:
oh no

Reply:
after first merge